src/dosbox.cpp

```diff
@@ -781,19 +781,22 @@ void DOSBOX_Init()
 	secprop->AddInitFunction(&DMA_Init);
 	secprop->AddInitFunction(&VGA_Init);
 	secprop->AddInitFunction(&KEYBOARD_Init);
+	secprop->AddInitFunction(&PCI_Init); // PCI bus
 
-	secprop=control->AddSection_prop("pci", &PCI_Init, false); // PCI bus
-
-	secprop->AddInitFunction(&VOODOO_Init, false);
+	secprop = control->AddSection_prop("voodoo", &VOODOO_Init, false);
 
 	const char* voodootypes[] = { "12mb", "4mb", "disabled", nullptr };
-	pstring = secprop->Add_string("voodoo", only_at_start, "12mb");
+	pstring = secprop->Add_string("voodoo_memsize", only_at_start, "12mb");
```

Comment:
I'd uppercase `MB` (megabyte is not `mb`) in the config description, but parse it in a case-insensitive manner.

Reply:
Dropping the 'mb' (or 'MB') postfix from these values, and going with integers to match the other `memsize` conf values (which take in integers).